spec/models/appointment_reminder_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe AppointmentReminder, type: :model do
+  describe "associations" do
+    it { should belong_to(:appointment) }
+    it { should belong_to(:patient) }
+    xit { should belong_to(:reminder_template) }
```

Comment:
This is temporarily disabled because I had issues getting the factory working for reasons I didn't figure out. I think it was somehow related to the use of the Experiment namespace. Any ideas?

Reply:
Can you paste the log of what was going on here?

Reply:
It was definitely a namespacing issue. I'm not sure if there's an easy way around that with this test method. I temporarily moved reminder template out of experiment and found that the test then fails with: 
```
Expected AppointmentReminder to have a belongs_to association called reminder_template (and for the record to fail validation if :reminder_template is unset; i.e., either the association should have been defined with `required: true`, or there should be a presence validation on :reminder_template)
```
So I think this is just a bad test and I should remove it.